File: src/androidTest/java/pt/joaomneto/titancompanion/TestCOH.java
package pt.joaomneto.titancompanion;


import android.support.test.filters.LargeTest;
import android.support.test.runner.AndroidJUnit4;

import org.junit.runner.RunWith;

import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook;

import static pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.CREATURE_OF_HAVOC;

@LargeTest
@RunWith(AndroidJUnit4.class)
public class TestCOH extends TestST {

    @Override
    protected FightingFantasyGamebook getGamebook() {
        return CREATURE_OF_HAVOC;
    }


}

File: src/androidTest/java/pt/joaomneto/titancompanion/TestCOH.kt
package pt.joaomneto.titancompanion

import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.filters.LargeTest
import org.junit.runner.RunWith
import pt.joaomneto.titancompanion.consts.FightingFantasyGamebook.CREATURE_OF_HAVOC

@LargeTest
@RunWith(AndroidJUnit4::class)
class TestCOH : TestST() {

    override val gamebook = CREATURE_OF_HAVOC
}
